tests/unit/test_metrics.py

```diff
@@ -153,6 +153,32 @@ def test_group_acceleration_data_by_time() -> None:
 )
 def test_compute_nonwear_value_per_axis(
     create_acceleration: pl.DataFrame, modifier: int, expected_result: int
+) -> None:
+    """Test the nonwear value per axis function."""
+    std_criteria = modifier
```

Comment:
Not sure this is a necessary alias, perhaps get rid of this and then just change the function call to `metrics._compute_nonwear_value_per_axis(
        acceleration_grouped["X"], std_criteria= modifier
    )`